Brewfile

```diff
@@ -14,5 +14,5 @@ brew 'gstreamer'
 brew 'pkgconfig'
 brew 'protobuf'
 brew 'protobuf-c'
-brew 'qt'
+brew 'qt@5'
```

Comment:
Your rebase made some duplicate commits (e.g. ce2759e). You'll want to do an interactive rebase and remove those.

Reply:
Fixed, thank you